architecture/core_concepts/routes.adoc

```diff
@@ -644,3 +670,34 @@ For example, a single route may belong to a `SLA=high` shard
 (but not `SLA=medium` or `SLA=low` shards),
 as well as a `geo=west` shard
 (but not a `geo=east` shard).
+
+For example, this allows the operator to partition the routes (via namespace/selectors)
+to be served by different router instances. And we could easily partition the
+routes in such a manner that the routers service overlapping sets (ala
```

Comment:
I know I wrote `ala` but that might possibly need some cleanup for docs -- maybe we can say 
`For example, deploying routers that service namespaces [A-J]*, [K-T]*  ... ` 
in a second second sentence.